getting-started/windows-calico/troubleshoot.md

```diff
@@ -104,3 +104,65 @@ If the error includes 'loading config file "<path-to-kubeconfig>"', follow the i
 #### Felix starts, but does not output logs
 
 By default, Felix waits to connect to the datastore before logging (in case the datastore configuration intentionally disables logging). To start logging at startup, update the [FELIX_LOGSEVERITYSCREEN environment variable]({{site.baseurl}}/reference/felix/configuration#general-configuration) to "info" or "debug" level.
+
+#### {{site.prodname}} BGP mode: connectivity issues, Linux calico/node pods report unready
+
+Check the detailed health output that shows which health check failed:
+
+```
+kubectl describe pod -n kube-system <calico-node-pod>
+```
+
+If the health check reports a BGP peer failure, check the IP address of the peer is either an
+expected IP of a node or an external BGP peer. If the IP of the failed peering is a Windows node:
+
+- Check that the node is up a reachable over IP
+- Check that the RemoteAccess service is installed and running:
+
+  ```powershell
+  PS C:\> Get-Service | ? Name -EQ RemoteAccess
+  ``` 
+- Check the logs for the confd service in the configured log directory for errors
+(default C:\TigeraCalico\logs).
```

Comment:
I wonder if we are at the point where we can add a variable for the Windows root dir name to _config.yml like we did with `site.prodnameWindows`: https://github.com/projectcalico/calico/blob/fc70a4f61ee13d8e63207bd778c20e8b1bcdc6f6/_config.yml#L21

Reply:
Added `rootDirWindows: C:\CalicoWindows`